python/Ganga/GPIDev/Adapters/IPrepareApp.py

```diff
@@ -45,7 +53,8 @@ def prepare(self, force=False):
         """
         Base class for all applications which can be placed into a prepared\
         state. 
-
+        Args:
+            force (bool) : ignored by the base class. Higher up forces the prepare function to be called no matter what
```

Comment:
No need to state that it's ignored by the base class. All subclasses must reimplement this function. Thedocstring should just describe what the argument is for:
```python
force (bool) : Force the prepare function to be called no matter what
```